gravitee-am-test/specs/gateway/remember-me.jest.spec.ts

```diff
@@ -140,9 +141,11 @@ describe('remember me', () => {
     });
 
     afterEach(async () => {
-      await deleteUser(domain.id, accessToken, user.id);
-      if (domain.id) {
-        await deleteDomain(domain.id, accessToken);
+      if (user?.id) {
+        await deleteUser(domain.id, accessToken, user.id);
+      }
+      if (domain?.id) {
+        await safeDeleteDomain(domain.id, accessToken);
       }
```

Comment:
suggestion: since the utility handles null:

```suggestion
      await safeDeleteDomain(domain?.id, accessToken);
}
```